core/qt-init.cpp

```diff
@@ -22,7 +22,10 @@ void init_qt_late()
 	QTextCodec::setCodecForLocale(QTextCodec::codecForMib(106));
 
 	QCoreApplication::setOrganizationName("Subsurface");
-	QCoreApplication::setOrganizationDomain("subsurface.hohndel.org");
+	QCoreApplication::setOrganizationDomain("subsurface-divelog.org");
```

Comment:
while this is the correct domain, I wonder if this will break things when upgrading. I.e., won't that cause the settings from previous versions to no longer be found?

Reply:
Right, I've changed it back.